benchmarks/runners/apfel_.py

```diff
@@ -280,9 +302,9 @@ def benchmark_nlo(self):
     # proj.benchmark_pto(0)
     # proj.benchmark_pto(1)
 
-    pos = BenchmarkPositivity()
-    pos.benchmark_pto(0)
-    pos.benchmark_pto(1)
+    # pos = BenchmarkPositivity()
+    # pos.benchmark_pto(0)
+    # pos.benchmark_pto(1)
```

Comment:
If possible, consider going in the direction of EKO's [`lha.py`](https://github.com/NNPDF/eko/blob/master/benchmarks/lha_paper_bench.py). I.e. use PyTest markers, instead of commented lines (such that we limit this kind of diffs)

Reply:
sure, let's keep this in mind; maybe we can do that in another PR, as we might want to introduce a similar workflow also here....